assets/theme-supplement.css

```diff
@@ -1632,3 +1632,33 @@ img {
 .sc-field-outline:focus-within {
```

Comment:
@aaronryematawaran Please, provide a new screenshot with the current changes applied, also test it with full width banners and images.